### DISCUSSION TITLE
Create a discord chat platform?

### DISCUSSION THREAD

**Discussion by webvs2** [preactjs/preact#4052] [General]
Would you consider it?

**Comment by Kanaye** [comment]
Preact has a slack worspace at: https://chat.preactjs.com/
So at least I wouldn't see a benefit in creating a second chat space.

**Comment by marvinhagemeister** [comment]
Sure, if a larger portion of users prefer discord over slack yeah. So far we've gotten like 1 or maybe 3 users over a year asking about discord and that's not enough to justify moving the whole community over to a different platform.
